apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -208,4 +210,18 @@ export class WidgetsController {
       success: true,
     };
   }
+
+  /*
+   * ValidationPipe convert boolean undefined params default (false)
+   * Therefore we need to get string and convert it to boolean
+   */
+  private initializeSeenParam(seen: string): boolean | null {
+    let isSeen: boolean = null;
+
+    if (seen) {
+      isSeen = seen == 'true';
```

Comment:
```suggestion
      isSeen = seen === 'true';
```

I think might be better to do strict check here